app/helpers/application_helper.rb

```diff
@@ -1,5 +1,5 @@
 module ApplicationHelper
-  def options_for_rating
-    [5, 4, 3, 2, 1].map { |n| [pluralize(n, "Star"), n] }
+  def options_for_rating(selected=nil)
+    options_for_select((1..5).map { |n| [pluralize(n, "Star"), n] }, selected)
```

Comment:
I would suggest using a full variable name here instead of `n` - this is just a personal style thing.